spark3/src/test/java/org/apache/iceberg/actions/TestCreateActions.java

```diff
@@ -67,6 +76,8 @@
       "PARTITIONED BY (id INT) STORED AS parquet LOCATION '%s'";
   private static final String CREATE_HIVE_PARQUET = "CREATE TABLE %s (data STRING) " +
       "PARTITIONED BY (id INT) STORED AS parquet";
+  private static final String CREATE_HIVE_PARQUET_UNPARTITIONED = "CREATE EXTERNAL TABLE %s (id INT, data STRING) " +
+          "STORED AS parquet LOCATION '%s'";
```

Comment:
Iceberg uses 2 spaces for regular formatting and 4 spaces for continued lines (like above). This line should be 4 instead of 8.

Reply:
Thanksfor lettting me know!!, I have fixed the indentations.